dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/VisitorLitmusPTX.java

```diff
@@ -210,7 +215,7 @@ public Object visitAtomConstant(LitmusPTXParser.AtomConstantContext ctx) {
         IOpBin op = ctx.operation().op;
         String mo = ctx.mo().content;
         String scope;
-        if (mo.equals(Tag.PTX.ACQ_REL) || mo.equals(Tag.PTX.RLX)) {
+        if (mo.equals(Tag.PTX.ACQ) || mo.equals(Tag.PTX.REL) || mo.equals(Tag.PTX.ACQ_REL) || mo.equals(Tag.PTX.RLX)) {
             scope = ctx.scope().content;
         } else {
             throw new ParsingException("Atom instruction doesn't support mo: " + mo);
```

Comment:
Why only read the scope if the mo is of a certain shape? If `ctx.scope().content` cannot throw a `NullPointerException`, then just do it before doing the mo-check. If it can throw the exception, just do it after the mo-check.